undertow/src/main/java/org/wildfly/extension/undertow/ServletContainerAdd.java

```diff
@@ -92,7 +92,8 @@ public void installRuntimeServices(OperationContext context, ModelNode model, St
         final ModelNode fileCacheTtlNode = ServletContainerDefinition.FILE_CACHE_TIME_TO_LIVE.resolveModelAttribute(context, model);
         final Integer fileCacheTimeToLive = fileCacheTtlNode.isDefined()  ? fileCacheTtlNode.asInt() : null;
         final int defaultCookieVersion = ServletContainerDefinition.DEFAULT_COOKIE_VERSION.resolveModelAttribute(context, model).asInt();
-        final boolean preservePathOnForward = Boolean.parseBoolean(System.getProperty("io.undertow.servlet.dispatch.preserve_path_of_forward", "false"));
+        final ModelNode preservePathOnForwardNode = ServletContainerDefinition.PRESERVE_PATH_ON_FORWARD.resolveModelAttribute(context, model);
+        final boolean preservePathOnForward = preservePathOnForwardNode.isDefined()? preservePathOnForwardNode.asBoolean():true;
```

Comment:
Please change the default value to false. As I'm putting in the analysis document, I'm choosing the default behavior of the property added to JBoss EAP to keep backwards compatibility with EAP previous versions

Reply:
Just note: this will lead in default behaviour changes between WildFly 15 (old behaviour), 16, 17 (new behaviour) and 18 (old behaviour again). But I agree with Flavia, that we want to keep old behaviour as default in upcoming version of JBoss EAP to stay consistent there.

Reply:
Thanks @fl4via. I pushed the change and also removed some redundant code form transformer test. Let me know if there's anything else I can do